api/database/factories/UserMocks.php

```diff
@@ -70,3 +70,16 @@
         'is_default' => $faker->boolean(),
     ];
 });
+
+$factory->define(\Spira\Rate\Model\Rating::class, function (\Faker\Generator $faker) {
+    return [
+        'rating_id' => $faker->uuid(),
+        'rating_value' => $faker->numberBetween(1, 10),
```

Comment:
The properties are polymorphic, so the properties should be `rateable_id` and `rateable_type`. The type should be pulled from a list of options that can be rateable, defined in the ratings model as static property.